WordPress/wordpress-playground
# Puzzle app improvements

## Motivation for the change, related issues

After releasing the beta version of the puzzle app, we got some feedback and want to address it.

### Feedback

The decorative image on the splash page doesn’t scale on different devices. On my iPhone 13 mini, it takes up 75% of the screen and overlaps with the text, making it more difficult to read. On a tablet, it’s cut off on the sides. Can we give it a fixed height of 50vh and scale automatically to ensure there’s enough space for the text above?
        To achieve that, we must replace the image with something that makes a better use of available space. I attached a new design ([Figma](https://href.li/?https://www.figma.com/design/Lf5SnXGegrD7avC8HMjFsm/Jarek%27s-Scratchpad?node-id=2778-14027&t=U8WURMkjXCfNM59a-4)). If we stick it to the left bottom edge, it should scale well. Please export in @2x or @3x to make sure it’s crisp on different devices. 👌
    It takes a moment for the app to recognize the puzzle. Can we add a [spinner](https://href.li/?https://wordpress.github.io/gutenberg/?path=/docs/components-spinner--docs) inside the camera button to let the user know something’s happening under the surface?
    It could be just me, but isn’t the puzzle piece backward compared to the PDF you attached? 🤔 Shall we flip it horizontally so the outline matches the puzzle shape?
    Can we reduce the top padding? I think it can be the same as the bottom one.
    The help text at the top needs to be wider. We should also tweak the line height to 30 (190%) to improve legibility.
    Let’s update the button color to use the new WordPress Blue (#3858E9 instead of #007CBA).

## Testing Instructions (or ideally a Blueprint) 

- [Start the puzzle app ](https://github.com/WordPress/wordpress-playground/blob/006cb0d5eab10d238388492b7a1fd332801a85d6/packages/playground/puzzle/README.md#L7)
- Confirm that the feedback was addressed
